spec/models/course_spec.rb

```diff
@@ -10,6 +10,8 @@
 
   describe "relationships" do
     it { should have_many :student_courses }
+    it { should have_many :students }
```

Comment:
We don't need to specify `have_many :students, through: :student_courses`?

Reply:
In tests, it's not required, but it could be added for clarity if we want